.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F680Feature request"
+about: Suggest a feature for EdgeX Framework
+labels: enhancement
+---
+<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅
+
+Oh hi there! 😄 
+
+To expedite issue processing please search open and closed issues before submitting a new one.
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->
+
+
+# 🚀 Feature Request
+
+### Relevant Package
+<!-- Can you pin-point one or more EdgeX services the are relevant for this feature request? -->
+<!-- ✍️edit: --> This feature request is for...
+
+
+### Description
+<!-- ✍️--> A clear and concise description of the problem or missing capability...
```

Comment:
As a matter of style I don't like sentences ending with ellipsis but it's not strictly wrong.